<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLServerForm</class>
 <widget class="QWidget" name="SQLServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="SetupGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QWidget" name="UserManager">
      <attribute name="icon">
       <iconset resource="../IDE.qrc">
        <normaloff>:/Menu/rc/Menu/user.png</normaloff>:/Menu/rc/Menu/user.png</iconset>
      </attribute>
      <attribute name="title">
       <string>UserManager</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QComboBox" name="Users"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Users_lineedit"/>
         </item>
         <item>
          <widget class="QPushButton" name="Search_Btn">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/find.png</normaloff>:/Menu/rc/Menu/find.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>RecordCount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="RecordCount">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Add_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Delete_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="Users_Table">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DeviceManager">
      <attribute name="icon">
       <iconset resource="../IDE.qrc">
        <normaloff>:/Page/rc/Page/manufacture.png</normaloff>:/Page/rc/Page/manufacture.png</iconset>
      </attribute>
      <attribute name="title">
       <string>DeviceManager</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QComboBox" name="Devices"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Devices_lineedit"/>
         </item>
         <item>
          <widget class="QPushButton" name="Device_Sreach_Btn">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/find.png</normaloff>:/Menu/rc/Menu/find.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>RecordCount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="DeviceRecordCount">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Device_Add_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Device_Del_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="Device_Table"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RcManager">
      <attribute name="icon">
       <iconset resource="../IDE.qrc">
        <normaloff>:/Page/rc/Page/ui.png</normaloff>:/Page/rc/Page/ui.png</iconset>
      </attribute>
      <attribute name="title">
       <string>RcManager</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QComboBox" name="Rc"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Rc_lineedit"/>
         </item>
         <item>
          <widget class="QPushButton" name="Rc_Sreach_Btn">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/find.png</normaloff>:/Menu/rc/Menu/find.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>RecordCount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="RcRecordCount">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Rc_Add_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Rc_Del_Record">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="Rc_Table"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabelExplorer">
      <attribute name="icon">
       <iconset resource="../IDE.qrc">
        <normaloff>:/Menu/rc/Menu/clearbytes.png</normaloff>:/Menu/rc/Menu/clearbytes.png</iconset>
      </attribute>
      <attribute name="title">
       <string>TabelExplorer</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTreeWidget" name="Sql_Table">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>  表名</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>RecordCount</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_19">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_20">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_21">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Menu/rc/Menu/refresh.png</normaloff>:/Menu/rc/Menu/refresh.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="TableContent"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
